src/csharp/Microsoft.Spark.Worker/Processor/BroadcastVariableProcessor.cs

```diff
@@ -62,15 +62,21 @@ internal BroadcastVariables Process(Stream stream)
                                 $"server {readBid} is different from the Broadcast Id received " +
                                 $"from the payload {bid}.");
                         }
+#pragma warning disable SYSLIB0011 // Type or member is obsolete
+                        // TODO: Replace BinaryFormatter with a new, secure serializer.
```

Comment:
Do you want to add this TODO to the other locations where you added `pragma warning disable SYSLIB0011` ?

Reply:
Added